run.sh

```diff
@@ -11,9 +11,12 @@ if [ "${ISISOLATED}" = true ]; then
 fi
 
 if [ -z "$TAG" ]; then
-	TAG="latest"
+    TAG="latest"
 fi
 
+UID="$(id -u "${USER}")"
```

Comment:
Are these environment variables needed in this script?

Reply:
right not anymore. I tried something else using gosu which needed them but couldn't make it work. I forgot to remove them thanks.